code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -802,11 +802,46 @@
 						continue
 					M.reagents.add_reagent("jenkem", 1)
 
+		if("hem")
+			message = "<b>[src]</b> hems."
```

Comment:
really?

Reply:
Never heard of this before, and thought it was some sort of weird furry fetish thing until Fox also said he doesn't understand it. Cough, faint, glare, etc. are all socials that humans typically use in everyday interaction. Never seen anyone "hem" before.